internal/docker/build.go

```diff
@@ -40,12 +36,19 @@ func Build(
 		"--label",
 		"org.opencontainers.image.source=https://github.com/bufbuild/plugins",
 		"--label",
+		fmt.Sprintf("org.opencontainers.image.created=%s", time.Now().UTC().Format(time.RFC3339)),
+		"--label",
 		fmt.Sprintf("org.opencontainers.image.description=%s", plugin.Description),
 		"--label",
 		fmt.Sprintf("org.opencontainers.image.licenses=%s", plugin.SPDXLicenseID),
+		"--label",
+		fmt.Sprintf("org.opencontainers.image.vendor=%s", "Buf Technologies, Inc."),
 		"--progress",
 		"plain",
 	}
+	if gitCommit := plugin.GitCommit(ctx); gitCommit != "" {
+		commonArgs = append(commonArgs, "--label", fmt.Sprintf("org.opencontainers.image.revision=%s", gitCommit))
```

Comment:
If there aren't uncommitted changes to the plugin's directory, we'll add the full git revision as an image label.